docs/program/python-venv.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Prepare your Python Virtual Environment"
+linkTitle: "Prepare your Python Virtualenv for Viam"
+weight: 50
+type: "docs"
+description: "Prepare your Python Virtual Environment to program machines with the Python SDK."
+images: ["/services/icons/sdk.svg"]
+tags: ["client", "sdk", "application", "sdk", "fleet", "program", "python", "venv"]
+---
+
+To manage Python packages for your Viam application, it is recommended that you use a virtual environment, or `venv`.
+By using a `venv`, you can install Python packages like Viam's client SDK just within the virtual environment, and not globally, which could cause conflicts with other projects or cause issues within your system.
+
+Follow this guide to set up a fresh virtual environment on your working computer and install the Python SDK as a requirement for your Viam client application.
+
+## Setup your project
+
+First, create a directory to house your project.
+For example, name your directory `viam-python`:
+
+```bash
+mkdir viam-python
+cd viam-python
+```
+
+## Create a Virtual Environment
+
+Now that you are in the project directory, create and activate a virtual environment for Python to run in.
+
+> **INFO**
+> Creating a virtual environment (`venv`) is important as it isolates this python environment from any other you might already have.
```

Comment:
We currently don't have this type of styling in our style guide, while we can add it we'd need to discuss that as a team first. 

Let's stick with an alert until then

Reply:
However, this also duplicates the info above. in the first paragraph on the page. I think we should not duplicate that.